guides/v2.1/template.md

```diff
@@ -13,31 +13,31 @@ To get started, edit your local version of this file, using the {% glossarytoolt
 
 Your contributions to our Docs and your experience with using Magento, are very valued and appreciated. Let us know if you have any questions!
 
-<h2 id="overview">Overview</h2>
+## Overview 
 PROVIDE OVERVIEW OF TOPIC HERE
 
-<h2 id="H2">HEADING 2</h2>
+## HEADING 2 
 PROVIDE TEXT HERE FOR FIRST SECTION.
 
 <div class="bs-callout bs-callout-info" id="info">
 
-  <p>INSERT NOTE TEXT HERE</a>.</p>
+  <p>INSERT NOTE TEXT HERE.</p>
 
 </div>
 
-<h3 id="H3">HEADING 3</h3>
+### HEADING 3 
 PROVIDE TEXT HERE FOR NEXT SECTION.
 
 TO HYPERLINK TO ANOTHER TOPIC... refer to the <a href="{{ page.baseurl }}/extension-dev-guide/bk-extension-dev-guide.html">NAME OF TOPIC OR BOOK</a>.
 
 
 
-<h2 id="H2">HEADING 2</h2>
+## HEADING 2 
 PROVIDE TEXT HERE FOR NEXT SECTION.
 
 ADD DIAGRAM OR ILLUSTRATION <p><img src="{{ site.baseurl }}/common/images/NAME_OF_IMAGEjpg" alt="HOVER TEXT HERE"></p>
```

Comment:
Change this link, and all links (unless they are within an HTML note), to markdown links.